src/plugins/circular_view/src/CircularView.cpp

```diff
@@ -1137,7 +1137,7 @@ qint64 CircularViewRenderArea::asinToPos(const qreal asin) const {
 QList<Annotation *> CircularViewRenderArea::findAnnotationsByCoord(const QPoint &coord) const {
     QList<Annotation *> res;
     QPoint cp(coord - QPoint(width() / 2, getCenterY()));
-    for (CircularAnnotationItem *item : circItems) {
+    for (CircularAnnotationItem *item : qAsConst(circItems)) {
```

Comment:
"circItems" is a QMap. I'm not sure how exactly items are received from the QMap during the iterating through it. If QMap::values() method is called implicitly than there will be problems, because this method returns a temporary QList.